tests/test_sft_trainer.py

```diff
@@ -122,6 +122,11 @@ def test_resume_training_from_checkpoint():
 
 
 def test_resume_training_from_checkpoint_with_flag_true():
```

Comment:
I'm not sure if this method is useful or if it;s doing the same as the one above.

Reply:
Yes, this test also checks if tuning is resuming from checkpoint but here I am marking `resume_from_checkpoint` flag as true, as to test the logic in the `sft_trainer.py` when flag value is True, though it's behavior would be similar to when value of `resume_from_checkpoint` is `None`.

Let me know, if you think we should remove this test.

Reply:
It's a bit redundant as resume_from_checkpoint=None creates the same behavior as resume_from_checkpoint=True but fine to leave for now